src/views/new-evaluation/newEval.js

```diff
@@ -1,38 +1,49 @@
 interview.formEventsSetup = function () {
 
-    const listener = function (e) {
+    const getNewEvaluationData = function (e) {
         e.preventDefault();
         const radioInputs = document.querySelectorAll('input');
         const selectElements = document.querySelectorAll('select');
         const textareaElements = document.querySelectorAll('textarea');
-    
+
         const obj = {};
         selectElements.forEach((item) => {
             obj[item.name] = item.value;
         });
-    
+
         textareaElements.forEach((item) => {
             obj[item.name] = item.value;
         });
-    
+
         for (let i = 0; i < radioInputs.length; i++) {
             const item = radioInputs.item(i);
-    
+
             if (item.checked || item.type === 'text' || item.type === 'date') {
                 obj[item.name] = item.value;
             }
         }
         console.log(obj);
-    
+
         // local storage
         let arr = [];
+        //verify if localStorage object contains data 
         if (localStorage.length !== 0) {
             arr = JSON.parse(localStorage.getItem('evaluations'));
         }
+        //add data from evaluation form in localStorage
         arr.push(obj);
         localStorage.setItem('evaluations', JSON.stringify(arr));
     };
+    let form;
```

Comment:
Same question as somewhere above: could you remind me what this part of the code is doing and why it was needed? :)